site/app/templates/autograding/TAResults.twig

```diff
@@ -82,6 +82,11 @@ $( document ).ready(function() {
                             {% endif %}
                         {% endfor %}
                         {% for file in uploaded_pdfs %}
+                            {% if file.encoded_name in annotated_file_names %}
+                                {% if can_download %}
+                                    <button class = 'btn btn-default key_to_click' onclick='downloadStudentAnnotations("{{ student_pdf_download_url }}?filename={{ file.name }}&path={{ file.path|url_encode }}", "{{file.path|url_encode}}", "annotated_pdfs")' aria-label="Download the file"> Download
+                                        <i class="fas fa-download" title="Download the file"></i></button>
+                                {% endif %}
                             {% if file.name in annotated_file_names %}
```

Comment:
With the changes, can this if statement ever be entered, or would it only enter the one above?

Reply:
Only the one above. This is why the View Popup button is gone. This has been fixed